index.html

```diff
@@ -14,252 +14,400 @@
-          </li>
+            <li class="elipse-name">
+              <img src="img/Ellipse 1 (1).png" class="elipsehtml" />
+              <p class="technology-language">HTML</p>
+            </li>
+            <li class="elipse-name">
+              <img src="img/Ellipse 1 (2).png" class="elipsecss" />
+              <p class="technology-language">CSS</p>
+            </li>
+          </ul>
 
-          <li class="major-lang">
-            <h5 class="about-h5">Skills</h5>
-            <div class="icon-right"></div>
-          </li>
-        </ul>
+          <ul class="frame">
+            <li class="about-framework">
+              <h5>Frameworks</h5>
+              <div class="icon-right"></div>
+            </li>
+            <li class="about-skill">
+              <h5>Skills</h5>
+              <div class="icon-right"></div>
+            </li>
+          </ul>
+        </div>
       </section>
-      <!-- form-section -->
-      <footer>
-        <section class="form-section">
-          <h3 class="contact-me">Contact me</h3>
-          <p class="apply">
-            If you have an application you are interested in developing, a
-            feature that you need built or a project that needs coding. I’d love
-            to help with it
-          </p>
-          <form action="https://formspree.io/f/mqknzyow" method="POST">
-            <input
-              type="text"
-              name="user_name"
-              placeholder="User name"
-              required
-              maxlength="30"
-            />
-            <input
-              type="email"
-              name="user_email"
-              placeholder="Email address"
-              required
-            />
-            <textarea
-              placeholder="Write your message here"
-              name="user_text"
-              required
-              maxlength="500"
-            ></textarea>
-            <button type="submit">Get It Touch</button>
-          </form>
-        </section>
-      </footer>
     </section>
 
+    <footer>
+      <section class="form-section">
+        <h3 class="contact-me">Contact me</h3>
+        <p class="apply">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form action="https://formspree.io/f/mqknzyow" method="POST">
+          <input
+            type="text"
+            name="user_name"
+            placeholder="Ismail Akinkunmi"
+            required
+            maxlength="30"
+          />
+          <input
+            type="email"
+            name="user_email"
+            placeholder="footer@gmail.com"
+            required
+          />
+          <textarea
+            placeholder="Write your message here"
+            name="user_text"
+            required
+            maxlength="500"
+          ></textarea>
+          <button type="submit">Get It Touch</button>
```

Comment:
### OPTIONAL

- [ ] Your button is not aligned like in the original. To stick to the design you can fix it.

| Your design | Original |
| ------------- |:-------------:|
| ![image](https://user-images.githubusercontent.com/38635158/144633139-a77f7e44-448c-468c-a39a-950a793fd28d.png)    | ![image](https://user-images.githubusercontent.com/38635158/144633391-179fb1cc-6cfe-4fc7-b18a-9e3bed0ad5c4.png)    |